sample/db/samples/tax_rates.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
-north_america = Spree::Zone.find_by!(name: "North America")
+begin
+  north_america = Spree::Zone.find_by!(name: "North America")
+rescue ActiveRecord::RecordNotFound
+  puts <<~TEXT
+    Couldn't find 'North America' zone. Did you run `rake db:seed` first?
+
+    That task will set up the countries, states and zones required for Spree.
```

Comment:
I wouldn't mention `Spree` here. Mauybe it's better to talk about a generic store: eg. `That task will set up the countries, states and zones required for your store.`